### ISSUE TITLE
MAS 1.4.3: Elements must have sufficient color contrast (.dirname)

### ISSUE THREAD

**Issue by InduPriya1805** [microsoft/monaco-editor#1749]
**Environment Details:​**
www.typescriptlang.org/​​
​​
**Browser Details: ​​​**
Microsoft Edge 44.18362.1.0, Google Chrome 75.0.3770.142​
​OS: Windows (Version 1903, Build: 18362.113)​​
​
**Repro Steps: ​​​**
1) Hit the URL "www.typescriptlang.org"​​
2) Navigate till "Playground" link and press enter​​
3) Tab till editor field and type any wrong code. ​
4) Press F8 or Shift+F8 to open the errors​
5) Verify the color contrast ratio for text "1 of 2 problems" that is present at the error

**Actual Result: ​​​**
Color contrast ratio for text "1 of 2 problems"  is 2.9:1 which is less than 4.5:1
​​
**Expected Result: ​​​**
Color contrast ratio for text "1 of 2 problems"  should be greater than or equal to  4.5:1
​​
User Impact: ​​​
Keyboard users will find difficult to work if focus order is not proper between the controls​
​
